src/navargus/__init__.py

```diff
@@ -56,6 +57,8 @@
 _client = None
 _config = None  # type: Configuration
 NOT_WHITESPACE = re.compile(r"[^\s]")
+NAV_SERIES = tuple(int(i) for i in _NAV_VERSION.split(".")[:2])
```

Comment:
I seem to recall there is a function doing this, hm.. no need hunting it down, but common problem so there *ought* to be something.

Reply:
I was thinking the same, but was too lazy to look for it ;-)